### ISSUE TITLE
Batch generation of invoices for all customers

### ISSUE THREAD

**Issue by Filechaser** [kevinpapst/kimai2#2178]
Dear Kevin Papst
When printing invoices a client has to be selected. 
We generate our invoices weekly. Each week we have to generate an invoice for each client. 
Can we somehow select ("All Clients") at once and print all invoices in one go?

Register Invoices --> Filter: Client --> "All Clients" or multi select?
Is this easy to implement or not possible?
It would improve the workflow by a lot.

Sincerely Filechaser

**Comment by kevinpapst** [comment]
Not possible and will not be implemented in Kimai core.

But you can semi-automate that via command line:
https://www.kimai.org/documentation/invoices.html#create-invoices-with-cronjobs

Or you ask via email for a quote for building such a feature as custom plugin.

**Comment by kevinpapst** [comment]
I changed my mind, as this was requested multiple times AND I found out that it was slightly easier to add than I expected.

New screen looks like this and allows to either skip the customer or select multiple.
Invoices will still be grouped by customer!

<img width="1275" alt="Bildschirmfoto 2021-03-26 um 14 51 47" src="https://user-images.githubusercontent.com/533162/112641595-cb9d4700-8e42-11eb-90a5-a9fd6ba9c550.png">
